lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -505,7 +505,7 @@ sub overview {
 
 sub latest {
     my ($self) = @_;
-    my %search_args;
+    my %search_args = (limit => 1);
```

Comment:
http://search.cpan.org/dist/DBIx-Class/lib/DBIx/Class/ResultSet.pm#first doesn't specify it would do a limit